MantidQt/CustomInterfaces/src/Indirect/Stretch.cpp

```diff
@@ -219,20 +200,35 @@ void Stretch::saveWorkspaces(const QString &fitWorkspace,
 
 /**
  * Handles the plotting of workspace post algorithm completion
- * @param betaWorkspace		:: The name of the beta workspace to plot
- * @param sigmaWorkspace	:: The name of the sigma workspace to plot
  */
-void Stretch::plotWorkspaces(const QString &betaWorkspace,
-                             const QString &sigmaWorkspace) {
+void Stretch::plotWorkspaces() {
+
+  // Check workspace exists
+  IndirectTab::checkADSForPlotSaveWorkspace(m_fitWorkspaceName, false);
+
+  WorkspaceGroup_sptr fitWorkspace;
+  fitWorkspace = AnalysisDataService::Instance().retrieveWS<WorkspaceGroup>(
+      m_fitWorkspaceName);
+
+  auto sigma = QString::fromStdString(fitWorkspace->getItem(0)->getName());
+  auto beta = QString::fromStdString(fitWorkspace->getItem(1)->getName());
+  if (sigma.right(5).compare("Sigma") == 0) {
+    if (beta.right(4).compare("Beta") == 0) {
```

Comment:
This if doesn't do anything is that correct? The `else` block below relates to `sigma.right(5).compare("Sigma") == 0`